projects/swimlane/ngx-charts/src/lib/common/timeline/timeline.component.ts

```diff
@@ -13,14 +13,8 @@ import {
 import { brushX } from 'd3-brush';
 import { scaleLinear, scaleTime, scalePoint } from 'd3-scale';
 import { select } from 'd3-selection';
-import { id } from '../..//utils/id';
-import { ViewDimensions } from '../types';
-
-export enum TimelineScaleType {
```

Comment:
I thought we could use directly `ScaleType` and remove this enum since there is a fallback defined in the `else {}` block.

We're getting type errors if we have two "different" `enums`.